packages/connect-plugin-ethereum/package.json

```diff
@@ -18,13 +18,14 @@
         "Ethereum"
     ],
     "sideEffects": false,
-    "publishConfig": {
-        "main": "lib/index.js"
-    },
     "main": "src/index.ts",
     "files": [
         "lib/"
     ],
+    "exports": {
+        "node": "./lib/index.js",
```

Comment:
node sure about syntax but I think there is something like `"commonjs": "./lib/index.js"` which may be better

Reply:
Yes, I am still polishing it. Getting some inspirations from https://nodejs.org/api/packages.html#package-entry-points